torchinductor/sizevars.py

```diff
@@ -58,6 +58,8 @@ def simplify(self, expr):
         return sympy.expand(expr).subs(self.replacements)
 
     def simplify_with_ranges(self, expr, var_ranges):
+        if isinstance(var_ranges, tuple):
```

Comment:
Move this after the docstring (python won't fill `__doc__` unless the docstring is first line)

Perhaps we should also `assert len(var_ranges)==1`